modules/rosa-autoscaling-nodes.adoc

```diff
@@ -0,0 +1,61 @@
+:_module-type: PROCEDURE
+//Specify the _module-type as either "CONCEPT, PROCEDURE, or REFERENCE"
+
+// Module included in the following assemblies:
+//
+// rosa-nodes/rosa-autoscale-nodes.adoc
+
+[id="rosa-autoscaling_{context}"]
+= Autoscaling nodes in a cluster
+
+[role="_abstract"]
+Configure autoscaling to increase the number of worker nodes that are available in a cluster. Autoscaling for worker nodes is configured in the machine pool definition.
+
+Autoscaling is dependent on the resource quotas that are set. If there are insufficient resources, worker nodes are removed from cluster autoscaling when pods fail. Verify resource quotas and request quota increases from the link:https://aws.amazon.com/console/[AWS console].
```

Comment:
If I am reading this correctly I don't think it makes sense. If there are insufficient resources and pods are failing, then the autoscaler will ADD nodes not remove.

Reply:
I removed that sentence altogether. I misunderstood something in the discussion in the Google Doc. Anyway, it should be correct now. Thanks!